WorkloadManagementSystem/Executor/InputData.py

```diff
@@ -93,16 +93,24 @@ def optimizeJob(self, jid, jobState):
       return self.setNextOptimizer()
 
     # Is there input data or not?
-    result = self._getInputs(jobState)
+    result = jobState.getInputData()
     if not result['OK']:
       self.jobLog.error("Cannot retrieve input data", result['Message'])
       return result
-    if not result['Value']:
+    inputData = result['Value']
+
+    result = self._getInputSandbox(jobState)
+    if not result['OK']:
+      self.jobLog.error("Cannot retrieve input sandbox", result['Message'])
+      return result
+    inputSandbox = result['Value']
+
+    if not inputData and not inputSandbox:
       self.jobLog.notice("No input data. Skipping.")
```

Comment:
Maybe just update this log here to 
```suggestion
      self.jobLog.notice("No input data nor LFN input sandboxes. Skipping.")
```

Reply:
changed, thanks !